docs/explanations/application-structure-explanation.md

```diff
@@ -35,14 +35,7 @@ All options are documented in our [configuration reference](../api-reference/con
 `src/operations.ts` is the _entrypoint_, where DBOS looks for your code.
 At startup, the DBOS runtime automatically loads all classes exported from this file, serving their endpoints and registering their decorated functions.
 More precisely, DBOS assumes your compiled code is exported from `dist/operations.js`, the default location to which `src/operations.ts` is compiled.
-If you're writing a small application, you can write all your code directly in this file.
-In a larger application, you can write your code wherever you want, but should use `src/operations.ts` as an index file, exporting code written elsewhere:
```

Comment:
I would keep this example as an alternative way to do it.  Both are equally viable.